exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/diagnosiskey/DiagnosisKeyService.java

```diff
@@ -65,7 +67,11 @@ private long getExportedKeyCount(List<TemporaryExposureKey> keys) {
                 requestKey.rollingStartIntervalNumber,
                 requestKey.rollingPeriod,
                 requestKey.visitedCountries,
-                Math.toIntExact(ChronoUnit.DAYS.between(symptomsOnset, utcDateOf10MinInterval(requestKey.rollingStartIntervalNumber))),
```

Comment:
Using the efgs mapper here seems a bit odd, but it's an identity mapping anyhow, right? If it's used for non-efgs keys as well, then maybe it should just be in the diagnosis key package.

Reply:
Yep, idea was to use same thing everywhere, e.g. in case we'll need to map our values to something else, but this is wrong place. I'll remove that.